tests/Profiling/ProfileTest.php

```diff
@@ -217,6 +218,176 @@ public static function formattedDataDataProvider(): \Generator
             ],
         ];
 
+        yield [
+            $event,
+            [
+                [
+                    'trace' => [
+                        [
+                            'file' => '/index.php',
```

Comment:
```suggestion
                            'file' => '/var/www/html/index.php',
```

and below, to actually test it 😬

Reply:
This one I don't get, I've duplicated the existing test case, to have one with and one without prefixes. Isn't that good enough for a test or did I misinterpret your comment?

(test case to sentry.io coming up!)

Reply:
Oh I got it, 🤦 yes you are totally right... made the input the same for both test cases now 😅